### PR TITLE
Fix #1806 Wrong Database properties encoding

### LINKED ISSUE #1806: Wrong Database properties encoding
JabRef 3.6dev--snapshot--2016-08-22--master--74afbb7
windows 10 10.0 amd64 
Java 1.8.0_101

Steps to reproduce:

1. File > Database properties > Database encoding > Big5 (was/is windows-1252)

### PR DESCRIPTION
**Pull request by stefan-kolb**
Introduced by #1790.

### CODE REVIEW

File: src/main/java/net/sf/jabref/gui/dbproperties/DatabasePropertiesDialog.java [hunk 1]
```diff
@@ -175,7 +176,12 @@ public void setVisible(boolean visible) {
     }
 
     private void setValues() {
-        encoding.setSelectedItem(panel.getBibDatabaseContext().getMetaData().getEncoding());
+        Optional<Charset> charset = panel.getBibDatabaseContext().getMetaData().getEncoding();
+        if (charset.isPresent()) {
+            encoding.setSelectedItem(charset.get());
+        } else {
+            encoding.setSelectedItem(StandardCharsets.UTF_8);
```
Comment (stefan-kolb) [inline src/main/java/net/sf/jabref/gui/dbproperties/DatabasePropertiesDialog.java]: Or should we use the Globals.getDefault encoding here?

Comment (matthiasgeiger) [inline src/main/java/net/sf/jabref/gui/dbproperties/DatabasePropertiesDialog.java]: Yes. Use `Globals.pref.getDefaultEncoding()`.

**Comment by oscargus** [comment]
I checked the rest of the code and in all other places it is correctly handled. I guess I just went for the errors and not actually searching for the use of `getEncoding()`...

**Comment by oscargus** [comment]
LGTM! :+1:

**Comment by matthiasgeiger** [comment]
:stuck_out_tongue_winking_eye:

**Pull request merged**